BDArmory/CounterMeasure/CMDropper.cs

```diff
@@ -246,6 +255,16 @@ void SetupCM()
                     }
                     resourceName = "CMDecoy";
                     break;
+
+                case "bubble":
+                    cmType = CountermeasureTypes.Bubbles;
+                    cmSound = SoundUtils.GetAudioClip("BDArmory/Sounds/smokeEject");
+                    resourceName = "CMSBT";
```

Comment:
What does the SBT part of CMSBT stand for?
Maybe this could use a more readable name?